rs/nns/handlers/root/interface/src/client.rs

```diff
@@ -220,6 +220,7 @@ impl SpyNnsRootCanisterClientReply {
                 reserved_cycles_limit: Some(candid::Nat::from(10_u32)),
                 wasm_memory_limit: Some(candid::Nat::from(11_u32)),
                 log_visibility: Some(LogVisibility::Controllers),
+                wasm_memory_threshold: Some(candid::Nat::from(0_u32)),
```

Comment:
Let's add a comment explaining the semantics of `wasm_memory_threshold` being zero?

Reply:
Done.